### PR TITLE
Issue#97 HouseCRUD. Added House method in yaml, DTO, Mappers, HouseAp…

### PR DESCRIPTION
**Pull request by Dandria1802**
…iServiceImpl

dev
## ZenHub

* [Main ZenHub ticket](https://app.zenhub.com/workspaces/home-project-5f7b77ff8db73a001cb17009)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: home-data/src/main/resources/application.properties [hunk 1]
```diff
@@ -4,4 +4,4 @@ spring.datasource.password=${DATASOURCE_PASSWORD:password}
 spring.datasource.driver-class-name=org.postgresql.Driver
 spring.jersey.type=filter
 server.port=${PORT:8080}
-spring.jpa.hibernate.ddl-auto=validate
+spring.jpa.hibernate.ddl-auto=validate
```
Comment (Romanov-Niko) [inline home-data/src/main/resources/application.properties]: We should leave a blank line at the end

File: home-data-migration/src/main/resources/db/changelog/0.0.1/Issue96.xml [hunk 2]
```diff
@@ -154,4 +154,4 @@
             <column name="permission_id" value="9"/>
         </insert>
     </changeSet>
-</databaseChangeLog>
+</databaseChangeLog>
```
Comment (Romanov-Niko) [inline home-data-migration/src/main/resources/db/changelog/0.0.1/Issue96.xml]: We should leave a blank line at the end

File: home-service/src/main/java/com/softserveinc/ita/homeproject/homeservice/dto/HouseDto.java [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+@NoArgsConstructor
+@AllArgsConstructor
+public class HouseDto extends BaseDto {
+
+    private Integer quantityFlat;
+
+    private String houseArea;
+
+    private Integer adjoiningArea;
+//    private Addresses addresses;
```
Comment (Romanov-Niko) [inline home-service/src/main/java/com/softserveinc/ita/homeproject/homeservice/dto/HouseDto.java]: Remove dead code please

File: home-service/src/main/java/com/softserveinc/ita/homeproject/homeservice/service/impl/HouseServiceImpl.java [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+
+    @Override
+    @Transactional
+    public HouseDto updateHouse(Long id, HouseDto updateHouseDto) {
+        if (housesRepository.findById(id).isPresent()) {
+
+            Houses fromDB = housesRepository.findById(id).get();
+//            if (updateHouseDto.getAddresses() != null) {
+//                fromDB.setAddress(updateHouseDto.getAddresses());
```
Comment (Romanov-Niko) [inline home-service/src/main/java/com/softserveinc/ita/homeproject/homeservice/service/impl/HouseServiceImpl.java:39]: Dead code

**Review by Romanov-Niko — APPROVED** [review]

File: home-service/src/main/java/com/softserveinc/ita/homeproject/homeservice/dto/HouseDto.java (hunk 3 above)
Comment (Dandria1802) [reply home-service/src/main/java/com/softserveinc/ita/homeproject/homeservice/dto/HouseDto.java]: Ok

File: home-service/src/main/java/com/softserveinc/ita/homeproject/homeservice/service/impl/HouseServiceImpl.java (hunk 4 above)
Comment (Dandria1802) [reply home-service/src/main/java/com/softserveinc/ita/homeproject/homeservice/service/impl/HouseServiceImpl.java:39]: Ok

File: home-data-migration/src/main/resources/db/changelog/0.0.1/Issue96.xml (hunk 2 above)
Comment (Dandria1802) [reply home-data-migration/src/main/resources/db/changelog/0.0.1/Issue96.xml]: Ok

File: home-data/src/main/resources/application.properties (hunk 1 above)
Comment (Dandria1802) [reply home-data/src/main/resources/application.properties]: Ok

Comment (PoLytvy) [reply home-data/src/main/resources/application.properties]: Do we need to use spring.jpa.hibernate.ddl-auto at all?

Comment (Dandria1802) [reply home-data/src/main/resources/application.properties]: I think yes. Why not?

Comment (PoLytvy) [reply home-data/src/main/resources/application.properties]: Yeah, figured it out. It can be here

**Pull request closed without merging**